test/relayHub/RelayHub.test.ts

```diff
@@ -0,0 +1,1199 @@
+import { ether } from '@openzeppelin/test-helpers';
```

Comment:
Maybe we could order the imports.

Reply:
I think this should be enforced/checked by eslint/prettier. I wouldn't do this manually.

Reply:
I'll create a related task to include an eslint rule for this.

Reply:
thank you :)